src/modules/packages/main.py

```diff
@@ -124,9 +124,6 @@ def run():
         run_operations(pkgman, entry)
 
     if libcalamares.globalstorage.contains("packageOperations"):
-        operations = libcalamares.globalstorage.value("packageOperations")
-
-        for entry in operations:
-            run_operations(pkgman, entry)
+        run_operations(libcalamares.globalstorage.value("packageOperations"))
```

Comment:
This won't work as long as run_operations requires a package manager as first argument.